packages/ipfs/README.md

```diff
@@ -942,6 +942,25 @@ const node = await IPFS.create({
 
 The code above assumes you are running a local `signaling server` on port `9090`. Provide the correct values accordingly.
 
+#### Is there an alternative to webrtc-star that offers a similar functionality?
+
+Yes, stardust! A WebSockets based transport that uses a Relay to route the messages. To enable it, you will need to execute its `signaling server` (available on [libp2p-stardust](https://github.com/libp2p/js-libp2p-stardust)) and include the correct `stardustServerId` as follows:
+
+```JavaScript
+const stardustServerId = 'QmY7t45zkBxzw5rtWQp3oAjzoJGTjh49f7LNz4YPBtTpqy'
+
+const node = await IPFS.create({
+  config: {
+    Addresses: { 
+      Swarm: [
+        `/ip4/0.0.0.0/tcp/5892/ws/p2p-stardust/p2p/${stardustServerId}`
```

Comment:
Just noticed that `Address.Swarm` notation is pretty different from one in webrtc-star and legacy ws-star where user just points at anonymous endpoint.

- Is this because the `stardust` protocol requires prior knowledge of `stardustServerId`?
- This is the main README, are we going to replace `/ip4/0.0.0.0/tcp/5892` with a real server?
  (`0.0.0.0` makes sense in examples where we run one on localhost, but here it may be confusing to users who try to copy&paste just to try it out)

Reply:
When the `multiaddrs` are added to libp2p peer for listening, `js-ipfs` adds a `/p2p/{own_peer_id}`. 

In `webrtc-star`, it gets the valid `multiaddr`, however `stardust` multiaddr to listen must be `/p2p/${stardustServerId}/p2p/{own_peer_id}`. This is because `stardust` server is a libp2p node, while `webrtc-star` is not a libp2p node.

I thought about creating a fixture with the `peer-id` first, so that we have always the same id and not need this change. However, I think that this is not the best solution. 

Anyway, the plan is to go as you asked. Once we have some servers deployed for experimenting, the server id issue will not be a problem anymore and we can replace this by the server address. The same will happen for `webrtc-star`.